vpc/service/allocate.go

```diff
@@ -412,3 +416,189 @@ func (vpcService *vpcService) ValidateAllocation(ctx context.Context, req *titus
 
 	return &titus.ValidationResponse{}, nil
 }
+
+type allocation struct {
+	id      string
+	region  string
+	account string
+	subnet  string
+}
+
+func (vpcService *vpcService) FixOldAllocations(ctx context.Context) error {
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
So, I think the way you want this migration to work:
1. Create static subnet CIDR reservations for all the existing V4 addresses
2. Change `getStaticInterface` to allocate an ENI with 1 IPv4 and 1 IPv6 address
3. Add a backfill loop

The backfill loop would (per non-IPv6 having allocation):
1. Allocate a static interface
2. Use that interface's IPv6 address to backfill the allocation
3. Delete the interface.

Reply:
converted to allocating an eni upon request